README.md

```diff
@@ -1,55 +1,94 @@
-- farming
-- mint (constant inflation)
-- claim
+<p align="center">
+Crescent Core - Expanding DeFi capabilities through InterBlockchain Technology<br/><br/>
 
-<!-- markdown-link-check-disable -->
-- see the [main](https://github.com/crescent-network/crescent/tree/main) branch for the latest 
-- see [releases](https://github.com/crescent-network/crescent/releases) for the latest release
+<a href="https://pkg.go.dev/badge/github.com/crescent-network/crescent">
+    <img src="https://pkg.go.dev/badge/github.com/crescent-network/crescent">
+</a>
+<a href="https://codecov.io/gh/crescent-network/crescent">
+    <img src="https://codecov.io/gh/crescent-network/crescent/branch/main/graph/badge.svg">
+</a>
+<a href="https://github.com/crescent-network/crescent/workflows/Tests/badge.svg">
+    <img src="https://github.com/crescent-network/crescent/actions/workflows/test.yml/badge.svg">
+</a>
 
-## Dependencies
+</p>
 
-If you haven't already, install Golang by following the [official docs](https://golang.org/doc/install). Make sure that your `GOPATH` and `GOBIN` environment variables are properly set up.
 
-This project uses customized cosmos-sdk, Please check the difference on [here](https://github.com/crescent-network/cosmos-sdk/compare/v0.44.5...v1.0.2-sdk-0.44.5).
+## What is Crescent?
 
-| Requirement           | Notes             |
-|-----------------------|-------------------|
-| Go version            | Go1.16 or higher  |
-| Cosmos SDK (custom)   | v1.0.2-sdk-0.44.5 |
+Crescent is a DeFi Hub that provides innovative and powerful tools that empower users’ digital assets for maximizing their financial returns while manage associated risks in the most efficient way. In the base layer, Crescent core has a DEX functionality equipped with several unique characteristics.
+
+- A Hybrid DEX : a hybrid system of orderbook and AMM
+- Tick System : standardization of order price
+- Batch Execution : all orders included in same block are fairly executed
+- Ranged Pools : liquidity pools providing liquidity within predefined price range
+- Optimized Liquidity Incentives Strategy
+- Synergy with Crescent Boost
 
 ## Installation
 
+### Use binaries
+
+This is the easiest way to get started. Download a pre-built binary for your operating system. You can find the latest binaries on the [releases](https://github.com/crescent-network/crescent/releases) page.
+
+### Build from source
+
+**Step 1. Install Golang**
+
+Go version [1.16](https://go.dev/doc/go1.16) or higher is required for Crescent Core.
+
+If you haven't already, install Go by following the [official docs](https://golang.org/doc/install). Make sure that your `GOPATH` and `GOBIN` environment variables are properly set up.
+
+**Step 2. Get Crescent Core source code**
+
+Use `git` to retrieve Crescent Core from the [official repo](https://github.com/crescent-network/crescent/) and checkout the `main` branch. This branch contains the latest stable release, which will install the `crescentd` binary.
```

Comment:
I don't think `main` branch contains a **stable** release.

Reply:
Let me change that to just `the latest release`.